spinalcordtoolbox/scripts/sct_deepseg.py

```diff
@@ -559,16 +557,16 @@ def main(argv: Sequence[str]):
         # Models can have multiple input images -- create 1 QC report per input image.
         if len(output_filenames) == len(input_filenames):
             iterator = zip(input_filenames, output_filenames, [None] * len(input_filenames), qc_seg)
-        # Special case: totalspineseg which outputs 4-5 files per 1 input file
-        elif arguments.task == 'totalspineseg':
-            # `-step1-only: 1`: Use the 4th image ([3]) which represents the step1 output
-            if getattr(arguments, "step1_only") == 1:
+        # Special case: spine which outputs 4-5 files per 1 input file
```

Comment:
Tiny nit: Make sure 4-5 is updated to 2-4 (or whatever the final amount of files ends up being).